docs/blog/write-the-docs-2018-kickoff.rst

```diff
@@ -0,0 +1,77 @@
+.. post:: Jan 20, 2018
+   :tags: community updates, core team
+   :author: Mikey Ariel
+
+Write the Docs 2018 Kick-Off
+============================
+
+Greetings, global documentarians!
+
+Mikey here, writing from the deep dark cold European winter, but looking forward to spring and summer, full of sunshine and docs!
+Following Eric's winter-warming `2017 Stats <http://www.writethedocs.org/blog/write-the-docs-2017-stats/>`_ post, I'd like to highlight some of the activities, changes, and experiments that you can expect in 2018.
+
+These efforts are a part of optimizing our work in supporting and growing the community, and the also aim to keep things fresh and exciting for our documentarians.
```

Comment:
`the also`.

Also this should be written one sentence per line, so I can comment on the proper sentence :)